sklearn/model_selection/_search.py

```diff
@@ -1411,7 +1411,24 @@ class RandomizedSearchCV(BaseSearchCV):
         A generator over parameter settings, constructed from
         param_distributions.
 
+
+    Examples
+    --------
+    >>> from sklearn import linear_model, datasets
```

Comment:
I would suggest to make the import on 2 lines and just import the required function/classes

```suggestion
    >>> from sklearn.datasets import load_iris
    >>> from sklearn.linear_model import LogisticRegression
```